winrt/docsrc/CanvasFontFace.xml

```diff
@@ -280,6 +286,33 @@ Licensed under the MIT License. See LICENSE.txt in the project root for license
     <member name="F:Microsoft.Graphics.Canvas.Text.CanvasUnicodeRange.Last">
       <summary>The last code point in the Unicode range.</summary>
     </member>
+    <member name="T:Microsoft.Graphics.Canvas.Text.CanvasGlyphMetrics">
```

Comment:
Please add an experimental tag to this new type:

<member name="T:Microsoft.Graphics.Canvas.Text.CanvasGlyphMetrics" Experimental="true">

This will warn people using the API that is is new, not yet had bake time, so might change in breaking ways depending on feedback etc.